CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.7.1
+
+## Minor
+
+* Packit will determine `upstream_project_url` from git remote if not specified in the config.
```

Comment:
looks like a bugfix to me, not a minor thing

Reply:
This is more a small feature to me, but I'll change that if you wish.

Reply:
Changed.